Gems/ROS2/Code/Source/Camera/CameraSensor.cpp

```diff
@@ -107,7 +111,18 @@ namespace ROS2
 
     void CameraSensor::SetupPasses()
     {
-        AZ_TracePrintf("CameraSensor", "Initializing pipeline for %s\n", m_cameraSensorDescription.m_cameraName.c_str());
+        bool allowModification = false;
+        auto* registry = AZ::SettingsRegistry::Get();
+        AZ_Assert(registry, "No Registry available");
+        if (registry)
+        {
+            registry->Get(allowModification, Internal::AllowCameraPipelineModificationKey);
+        }
+        AZ_TracePrintf(
```

Comment:
`AZ_TracePrintf` is deprecated, use `AZ_Trace` instead
```suggestion
        AZ_Trace(
```